.travis.yml

```diff
@@ -19,10 +19,12 @@ matrix:
     - gemfile: gemfiles/Gemfile.public_suffix_2
       rvm: 2.5.5
   allow_failures:
+    - rvm: jruby-9.0.5.0
```

Comment:
This version of jruby doesn't seem to work with java 11, which is the default on Travis now. I'd therefore propose to drop it as it is officially unsupported.

Reply:
@dentarg Your call..

Reply:
I think we should just drop it. Allow failures makes sense if you expect it might sometimes pass. If you expect it to _always_ fail, it doesn't offer much value to run the test at all.